mythril/ether/soliditycontract.py

```diff
@@ -102,6 +102,13 @@ def get_source_info(self, address):
 
         index = helper.get_instruction_index(self.disassembly.instruction_list, address)
 
+        if index is None:
+            # File index for instruction address not found. But is there only *one* source file?
+            file_idx_set = {m.solidity_file_idx for m in self.mappings}
+            if len(file_idx_set) == 1:
+                # Yes, so use the single file mentioned.
+                index = file_idx_set.pop()
```

Comment:
the index here means instruction index.

Reply:
I believe it is  a _file_ index, not an instruction index. Right?

Reply:
Hmm. I suppose we could look for an instruction that has the right file index, and since there is only one file index, it should be easy to find. But all of this feels a little convoluted because were looking for an instruction index just to be able to reuse the instruction-index-based code below. 

Again, if you can think of a better way, that'd be perfect